front-end/src/screens/BountyPost/index.tsx

```diff
@@ -3,15 +3,16 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import React from 'react';
-import { useParams } from 'react-router-dom';
+import { useRouter } from 'src/hooks';
 
 import Post from '../../components/Post/Post';
 import { useBountyPostAndCommentsQuery } from '../../generated/graphql';
 import FilteredError from '../../ui-components/FilteredError';
 import Loader from '../../ui-components/Loader';
 
 export default () => {
-	const { id } = useParams();
+	const { query } = useRouter();
```

Comment:
The reason I did all these changes is because TS was complaining that "`id` can't be found in type `{}`".
The obvious problem here is the typing that is wrong. I kind of when around it by using a hook that was there, and type correctly what I received with `as string`.

Another, maybe better way would be to keep using the `useParams()` but type it as `as Record<string, string>`. Up to you what you prefer.